### ISSUE TITLE
test_ssl fails after 2019-09-03

### ISSUE THREAD

**Issue by bmwiedemann** [csernazs/pytest-httpserver#21]
While working on reproducible builds for openSUSE, I found that
our python-pytest-httpserver package fails to build 2019-09-04 and later
because `tests/assets/rootCA.crt` expired.
additionally `tests/assets/server.crt` will expire 2020-01-16

Either those certs should be re-generated as part of the test or expiry should be > 2100

**Comment by bmwiedemann** [comment]
Steps to reproduce on openSUSE or Debian:
```bash
sudo apt install osc qemu-kvm
osc checkout openSUSE:Factory/python-pytest-httpserver && cd $_
osc build --build-opt=--vm-custom-opt="-rtc base=2019-09-04T00:00:00" --vm-type=kvm --noservice --clean
```

**Comment by csernazs** [comment]
Thanks for the early warning! This was my mistake to specify 365 days explicitly in the Makefile in the assets directory.
While your PR looks ok for me, I think it is better to update the Makefile instead with the increased expiration, so I would go ahead with my PR.

**Comment by csernazs** [comment]
I have released 0.3.4 to pypi which now contains this fix. Thanks for the report again.

**Comment by bmwiedemann** [comment]
tested 0.3.4 - works fine. I noticed 2 things though:
https://github.com/csernazs/pytest-httpserver/releases does not mention anything about the certs in the release notes
and the directory name changed from pytest_ to pytest-

**Comment by csernazs** [comment]
CHANGES.rst has been added recently and it is generated by reno (https://docs.openstack.org/reno/latest/) but as reno uses git tags to label the releases, the tag must be available before the CHANGES.rst is updated. This is the reason why it is not included in the tar.gz on the github but included in the release available on pypi (as sdist is generated with the updated CHANGES.rst file).
This is a  known issue for me but currently it iseems a chicken and egg problem and I have no idea how to solve this. One way would be changing the git tag after generating the changelog but I', a bit reluctant about this.

I think the _ and - thing is related to the setuptools version I used and sometimes I'm doing it from the venv, sometimes I'm using the setuptools available on my system. I'll have a look into this later.
